.github/workflows/python-publish.yml

```diff
@@ -27,4 +27,4 @@ jobs:
           FLIT_USERNAME: ${{ secrets.PYPI_USERNAME }}
           FLIT_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         run: |
-          flit publish
+          flit publish
```

Comment:
not sure why this file is even touched in this PR, not a big deal, just a curiosity.

Reply:
Sure, will clean this up